core/app/assets/stylesheets/glass/layout/_pages.scss

```diff
@@ -0,0 +1,539 @@
+            display: none;
+          }
+        }
+        .page-title {
+          @include ellipsis(100%);
+          @include media-breakpoint-up(md) {
+            padding-right: 90px;
+          }
+          display: flex;
+          align-items: center;
+          align-content: center;
+          padding: 10px 0;
+          padding-right: 30px;
+          transition: 0;
+          color: $gray-6;
+          .title {
+            @include ellipsis(auto);
+          }
+          .label {
+            @include media-breakpoint-down(sm) {
+              display: none;
+            }
+            flex-shrink: 0;
+            margin-left: .5rem;
+            padding: 2px 8px;
+            font-weight: 600;
+            text-transform: capitalize;
+            background: $gray-3;
+            &:first-of-type {
+              margin-left: 1rem;
+            }
+          }
+        }
+        .attribute {
+          padding-right: 60px;
+        }
+        .date {
+          display: block;
+          flex-shrink: 0;
+          margin-left: auto;
+          font-size: .75rem;
+          text-align: right;
+          color: $gray-5;
+          @include media-breakpoint-down(sm) {
+            display: none;
+          }
+        }
+        .actions {
+          display: none;
+          align-items: center;
+          flex-shrink: 0;
+          margin-left: auto;
+          text-align: right;
+          @include media-breakpoint-down(sm) {
+            display: block;
+          }
+          &-visible {
+            display: flex;
+          }
+          &-tight-right {
+            right: -10px;
+          }
+          .circle-icon {
+            line-height: 1.6;
+            color: $cms-color;
+            border-color: $cms-color;
+            &:link,
+            &:visited {
+              color: $cms-color;
+            }
+            &:hover,
+            &:focus {
+              color: $white;
+              border-color: $cms-dark;
+              background: $cms-dark;
+            }
+            .icon-close {
+              font-size: .75rem;
+            }
+          }
+          @include media-breakpoint-up(md) {
+            .circle-icon {
+              color: $white;
+              border-color: $white;
+              &:link,
+              &:visited {
+                color: $white;
+              }
+            }
+          }
+          span {
+            display: inline-block;
+            width: 30px;
+            margin-left: 5px;
+            text-align: center;
+          }
+        }
+      }
+    }
+    /* nested ul */
+    ul {
+      margin: 0 0 0 0;
+      padding: 0;
+      list-style: outside none none;
+      &:empty {
+        margin: 0;
+      }
+      li {
+        margin-bottom: 0;
+        margin-left: 30px;
+        .page-row {
+          background: $white;
+        }
+        &:last-child {
+          .page-row {
+            margin-bottom: 4px;
+          }
+        }
+      }
+      li .page-row {
```

Comment:
I'm not sure about this part, it looks like the nested `li .page-row` level is hardcoded. What happens after 6 levels ?